CHANGELOG.md

```diff
@@ -176,6 +176,7 @@ For details about compatibility between different releases, see the **Commitment
 ### Added
 
 - Add more specific rights for OAuth clients.
+- Fallback `version_ids` to load end device profile from DR when importing a device in the Console.
```

Comment:
I would phrase this as `Fallback end device version identifiers to be used while importing end devices using the Console`.